source/content/guides/multisite/05-debug.md

```diff
@@ -10,36 +10,44 @@ type: guide
 cms: "WordPress"
 categories: [troubleshoot]
 tags: [multisite]
-pagination:
-    provider: data.multisitepages
-use:
-    - multisitepages
 permalink: docs/guides/multisite/debug/
-previousurl: guides/multisite/workflows/
 editpath: multisite/05-debug.md
 image: multisite
 getfeedbackform: default
 ---
+
 ## WP-CLI Tips and Tricks
 
 ### Managing Site Networks
+
 The `wp site *` class of WP-CLI commands ([full documentation](https://developer.wordpress.org/cli/commands/site/)) for managing the sites installed on your network. Here are some helpful ones:
 
 - `wp site create` - Create a new site on the network.
 - `wp site list` - See all available sites on the network.
 - `wp site empty` - Clear a site of its posts and comments, while retaining options, users and other configuration details.
 
 ### Updating the Database
+
 WordPress sometimes includes database schema changes in releases. When you update WordPress to the latest version, you might see a notification in the WordPress dashboard to run the database update process. Use `wp core update-db --network` ([full documentation](https://developer.wordpress.org/cli/commands/core/update-db/)) to run the database upgrade procedure across all sites on your Site Network.
 
 ### Adding and removing super admins
+
 “Super admin” is a special designation for select users on a WordPress Site Network. While a user's role may vary between sites on the network, any user can be a super admin which gives them unrestricted access to every site on the network.
 
 Use `wp super-admin list` to list current super admins, and `wp super-admin add` or `wp super-admin remove` to add or remove them, respectively.
 
 ## Troubleshooting
 
+### Error: Cookies are blocked or not supported by your browser.
+
+You must enable cookies for WordPress multisite users to log in to their admin interface. Users may need to add this in their `wp-config.php` file:
+
+```php:title=wp-config.php
+define('COOKIE_DOMAIN', $_SERVER['HTTP_HOST'] );
+```
```

Comment:
For technical review, this is the relevant section.

Reply:
I'm not sure why this is under multisite, this occurs on regular blogs as well.

I think the terms here aren't the most clear either. The "You" I think of as a site developer, they don't enable cookies, it is the individual "multisite users" logging in, who need to enable cookies.  "Users" don't have a php file to update, this is a change that the site developer makes.

Lastly, I'm not sure when or why this cookie domain change would be used. I don't have this and things worked fine. 

Is this a second line of trouble shooting if the user has cookies enabled but the site still doesn't work? Does this only work for multisite or could we use this code on a single site fix.

I could see this code as a possible remedy for a challenge that can arise with using custom domains on sub-sites that don't match the source site's domain but I think more context would be needed.

Reply:
CC @panth-jrefuerzo

Reply:
Hi @jrastaban @alexfornuto , we stumbled upon that error while working with a customer on this chat 51746600137739, then we tried the suggestions in here https://wordpress.stackexchange.com/questions/166181/cant-log-in-error-cookies-are-blocked-or-not-supported-by-your-browser-you and it seemed to work. I was helping out @panth-jrefuerzo in doing his first docs issue so asked him to open up this thread. We tried it with the customer that has a multisite but if it also works on non-MU site, would it be a good addition to the troubleshooting section and which part of the docs would this troubleshooting fit to? Thanks!

Reply:
Pinged'd @jrastaban on Slack to take another look.

Reply:
OK, I've added this note as part of our basic troubleshooting doc in progress (commit 7ad2c4959c813f6663c7cf6723aee16bb2ad375e). Would this be acceptable to everyone involved?